### PR TITLE
Migrating publishing to GitHub actions.

### PR DESCRIPTION
**Pull request by Tapchicoma**

### CODE REVIEW

File: .github/workflows/new-release.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        with:
+          path: ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-wrapper-${{ hashFiles('**/gradlew') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-wrapper-
+      - name: Publish plugin
+        env:
+          GRADLE_PUBLISH_KEY: {{ secrets.GradlePublishKey }}
+          GRADLE_PUBLISH_SECRET: {{ secrets.GradlePublishSecret }}
+          GITHUB_KEY: {{ secrets.GithubKey }}
```
Comment (Tapchicoma) [inline .github/workflows/new-release.yml:36]: @JLLeitschuh please check here secrets names

Comment (JLLeitschuh) [reply .github/workflows/new-release.yml:36]: Should be `GRADLE_PUBLISH_KEY` and `GRADLE_PUBLISH_SECRET`. I never setup a GitHub key. Isn't there one that's implicit in the environment?

Comment (Tapchicoma) [reply .github/workflows/new-release.yml:36]: I don't see anything related to `GITHUB_KEY` here: https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables#default-environment-variables

Comment (Tapchicoma) [reply .github/workflows/new-release.yml]: I've used proper secret name for `GRADLE_PUBLISH_KEY` and `GRADLE_PUBLISH_SECRET` via force-push.

**Review by JLLeitschuh — APPROVED** [review]
LGTM!

**Pull request merged**